CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 3.19)
 
 project(finufft VERSION 2.2.0 LANGUAGES C CXX)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
If we use this here, we can remove the C++17 flags everywhere else. Not sure if this is worth it ... just a suggestion.

Reply:
It is better to set it by target as this will influence projects that include this as a dependency.